app/src/main/java/com/hyperion/ths/marvel_03/ui/favorite/ItemFavoriteFragmentViewModel.java

```diff
@@ -0,0 +1,66 @@
+package com.hyperion.ths.marvel_03.ui.favorite;
+
+import android.databinding.BaseObservable;
+import android.view.View;
+import com.hyperion.ths.marvel_03.data.model.Hero;
+import com.hyperion.ths.marvel_03.ui.BaseRecyclerView;
+import com.hyperion.ths.marvel_03.utils.Constant;
+
+/**
+ * Created by ths on 06/06/2017.
+ */
+
+public class ItemFavoriteFragmentViewModel extends BaseObservable {
+    private Hero mHero;
+    private BaseRecyclerView.OnRecyclerViewItemClickListener<Hero> mItemClickListener;
+    private BaseRecyclerView.OnItemButtonClickListener<Hero> mButtonClickListener;
+    private boolean isFavorite;
```

Comment:
WARNING: [AvoidFieldNameMatchingMethodName] Field isFavorite has the same name as a method
https://pmd.github.io/pmd-5.5.1/pmd-java/rules/java/naming.html#AvoidFieldNameMatchingMethodName